gapic/src/main/com/google/gapid/models/Profile.java

```diff
@@ -161,6 +163,19 @@ protected void onUiThread(CommandStream.Node node) {
     });
   }
 
+  public Experiments getExperiments() {
+    return experiments;
+  }
+
+  public void setExperiments(Experiments experiments) {
+    this.experiments = experiments;
+  }
+
+  public void reload() {
+    // Melih TODO: Add experiments to source and set this to false
```

Comment:
Do this now. Simply put it in the class below, fix the equals and hashcode and you should be good to go. Then, in the `setExperiments` method update the source. Take a look at `updateSemantics` in the `Geometries` model for an example.